opensquirrel/ir.py

```diff
@@ -257,10 +256,10 @@ def parse(cls, axis: AxisLike) -> NDArray[np.float64]:
         if len(axis) != 3:
             msg = f"axis requires an ArrayLike of length 3, but received an ArrayLike of length {len(axis)}"
             raise ValueError(msg)
-        if np.any(axis != 0).item():
-            return axis
-        msg = "axis requires at least one element to be non-zero"
-        raise ValueError(msg)
+        if np.all(axis == 0).item():
```

Comment:
Why do we need the `.item()` here, @GuyPuts?

```suggestion
        if np.all(axis == 0):
```